bin/fk.sh

```diff
@@ -35,7 +35,11 @@ fpr()
       # assuming ssh remote since `fclone` clone from ssh remote by default
       repoName=$(git config --get remote.origin.url | xargs basename | cut -d'.' -f1)
       ownerName=$(git config --get remote.origin.url | cut -d'/' -f1 | cut -d':' -f2)
-      xdg-open https://github.com/$ownerName/$repoName/pull/new/$curBranch &> /dev/null
+      if curl -s https://github.com/$ownerName/$repoName/pull/$curBranch | grep -o "href=\".*\"" | grep -v new; then
```

Comment:
Why can't we let the browser redirect?

Reply:
It can't because looks like a bug on GitHub side since the redirect depends on the branch name. 

Suppose I have a branch called `az.foo`, github will redirect me to `https://github.com/<org>/<repo>/pull/new/az.foo`

But if the branch name is simply called `foo` then the github will redirect me to not found. 

The `if` condition will check the redirect link github send back, There are two cases:
1.  if it's a 404 it will fail the second `grep` which cause it to open the link to open new PR in the else case. 
2. if the redirect link contain word `new` it will fail the third `grep` and also open a new PR

Reply:
Actually now I realise i can probably just combine the second and third `grep`

Reply:
wtf...

Reply:
Well I guess the better solution is to use the GitHub API, but then that means i need to handle json which means writing ruby ....